ices/83394.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+rustc -C embed-bitcode=no -C codegen-units=1 -C debuginfo=2 --crate-type lib - <<'EOF'
```

Comment:
Why don't you just use `#![crate_type]` here?

Reply:
Resolved in ff2dfb5.